ui/app/common/patient-search/views/patientsList.html

```diff
@@ -76,21 +75,18 @@
             <ul>
                 <li ng-repeat="patient in search.visiblePatients" class="active-patient" ng-click="forwardPatient(patient)">
                     <img class="smallImages" ng-src="{{::patient.image}}" fallback-src='../images/blank-user.png'>
-
-                <div class="patient-name">{{::patient.name}}</div>
-                <div class="patient-id">{{::patient.identifier}}</div>
-                <i class="ipd-indication fa fa-bed" ng-if="::(patient.hasBeenAdmitted===true||patient.hasBeenAdmitted==='true')"></i>
-            </li>
+                    <div class="patient-name">{{::patient.name}}</div>
+                    <div class="patient-id">{{::patient.identifier}}</div>
+                    <i class="ipd-indication fa fa-bed" ng-if="::(patient.hasBeenAdmitted===true||patient.hasBeenAdmitted==='true')"></i>
+                </li>
             </ul>
         </resize>
         <div class="note-container" ng-if="search.noResultsMessage">
             <div class="note warning">
                 <div class="text">
-                    <p class="ng-binding">{{::search.noResultsMessage | translate}}</p>
+                    <p class="ng-binding">{{::search.noResultsMessage}}</p>
```

Comment:
We need |translate here. Is this done by mistake?

Reply:
By mistakenly removed it.